### PR TITLE
add lazy evaluation

### PR DESCRIPTION
**Pull request by lukewestby**
For #1

Thanks for getting back to me so quickly on this!

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -4,13 +4,23 @@ import {expect} from 'chai';
...
   });
-  it ('should not return the element when the predicate fails', () => {
-    expect(renderIf(false)('foobar')).to.be.eql(false);
+  describe('lazy', () => {
+    it('should return the result of the thunk when the predicate passes', () => {
+      expect(renderIf(true)(() => 'foobar')).to.be.eql('foobar');
+    });
+    it('should not return the result of the thunk when the predicate fails', () => {
+      expect(renderIf(false)(() => 'foobar')).to.be.eql(null);
+    });
```
Comment (ajwhite) [inline test.js]: Yay, ty for units :smile:

**Comment by ajwhite** [comment]
Ah, just realized I don't have Travis running at the moment for PR statuses. This looks good though, and if anything comes up, I'll follow up with a PR. I'll aim to tag and release this evening or tomorrow.

Thanks for the contribution @lukewestby!

File: test.js (hunk 1 above)
Comment (lukewestby) [inline test.js]: Wouldn't dream of submitting a PR without them! :smile_cat:

**Comment by lukewestby** [comment]
Hey @ajwhite, was this update published to npm?

**Comment by ajwhite** [comment]
Ooh.. you know, I don't think so. Doing that now, thanks for bumping

**Comment by ajwhite** [comment]
[0.0.5 bumped](https://github.com/ajwhite/render-if/commit/c981ab31d16db99d0a0322d83ffcc1ea46838b3e) and is published :+1:

**Comment by ajwhite** [comment]
[0.1.0](https://github.com/ajwhite/render-if/commit/03fd619253315e78bbdb8508e9cfa8297bfec0cc) bumped and published :+1:

**Comment by lukewestby** [comment]
Thank you!

**Pull request merged**